models/ssh_key.go

```diff
@@ -106,6 +106,8 @@ func extractTypeFromBase64Key(key string) (string, error) {
 func parseKeyString(content string) (string, error) {
 	// Transform all legal line endings to a single "\n".
 	content = strings.NewReplacer("\r\n", "\n", "\r", "\n").Replace(content)
+	// remove trailing newline (and beginning spaces too)
+	content = strings.TrimSpace(content)
```

Comment:
Likewise, it can be merged with the above line like so,

    content = strings.NewReplacer("\r\n", "\n", "\r", "\n").Replace(content).TrimSpace(content)